ui/client/components/MenuBar.tsx

```diff
@@ -60,6 +60,10 @@ type Props = {
 
 const Spacer = () => <Flex flex={1}/>
 
+function NussknackerLogo() {
+  return <img src={absoluteBePath("/assets/img/nussknacker-logo.svg")} className={"navbar-brand-logo"}/>;
```

Comment:
We use sth like `UrlUtils.absoluteBePath` which is easy to test on dev setup - because this asset is taken from BE instead of FE. Also you should move this asset on BE side to /web/static/assets.